torchvision/csrc/cpu/vision_cpu.h

```diff
@@ -2,39 +2,39 @@
 #include <torch/extension.h>
 #include "../macros.h"
 
-VISION_API at::Tensor DeformConv2d_forward_cpu(
-    const at::Tensor& input,
-    const at::Tensor& weight,
-    const at::Tensor& offset,
-    const at::Tensor& mask,
-    const at::Tensor& bias,
+VISION_API at::Tensor deform_conv2d_forward_cpu(
+    const at::Tensor& input_param,
+    const at::Tensor& weight_param,
+    const at::Tensor& offset_param,
+    const at::Tensor& mask_param,
+    const at::Tensor& bias_param,
     int64_t stride_h,
     int64_t stride_w,
     int64_t pad_h,
     int64_t pad_w,
-    int64_t dilation_h,
-    int64_t dilation_w,
-    int64_t groups,
-    int64_t deformable_groups,
+    int64_t dil_h,
+    int64_t dil_w,
+    int64_t n_weight_grps,
+    int64_t n_offset_grps,
```

Comment:
weren't the other names more informative?

Reply:
I have it on the TODO of this PR to drop the params (see description). Here I am flagging that the master branch uses different name across the headers and the cpp files. To facilitate the code review, I chose to do minimal changes until we agree on the basics.